evo1/seq/overworld.py

```diff
@@ -19,44 +20,27 @@ class OverworldGliFarm(SeqATBmove2D):
     def __init__(
         self,
         name: str,
-        coords: List[Vec2],
+        coords: list[Vec2],
         goal: FarmingGoal = None,
         precision: float = 0.2,
     ):
         super().__init__(name=name, coords=coords, goal=goal, precision=precision)
-        self.can_manip = True
-        self.started_manip = False
-        self.manipulated_enc: Optional[EncounterID] = None
+        self.manip_state = self._MANIP_FSM.CAN_MANIP
+        self.manipulated_enc: Optional[Encounter] = None
 
     def reset(self) -> None:
-        self.can_manip = True
-        self.started_manip = False
+        self.manip_state = self._MANIP_FSM.CAN_MANIP
         self.manipulated_enc = None
         super().reset()
 
     _CHEST_LOCATION = Vec2(84, 46)
     _CHEST_RNG_ADVANCE = 66  # Ticks the rng forward 66 steps
     _GLI_PER_ENEMY = 50
 
-    def _get_number_of_combatants(self, encounter: EncounterID) -> int:
```

Comment:
No need for this function, we can just get the combatants directly from the `Encounter` class.